configure.ac

```diff
@@ -93,9 +93,68 @@ AC_CANONICAL_HOST
 AC_SUBST(build)
 AC_SUBST(host)
 
+AS_VAR_IF([cross_compiling], [maybe],
```

Comment:
```suggestion
AS_VAR_IF([cross-compiling], [maybe],
```

Reply:
``AS_VAR_IF`` expects the name of a shell variable. ``cross-compiling`` is not a valid variable name.